app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
+//= link_directory ../stylesheets .css
+//= link_tree ../builds
```

Comment:
- [ ] When I try to start the application using `./bin/dev`, I get this error:

![Screenshot8](https://user-images.githubusercontent.com/77106962/224375118-b467ae2e-fe82-492e-affa-5dc089cce540.png)

#### The application doesn't have an `images` directory in the `app/assets` directory. To fix it, I would advise you to either delete this line of code `//= link_tree ../images` or create an empty `images` directory in the `app/assets` directory. 👍 😎